spec/enumerables_spec.rb

```diff
@@ -0,0 +1,196 @@
+# rubocop: disable Layout/LineLength
+
+require_relative '../lib/enumerables'
+
+describe '#my_each' do
+  it 'return enumerator if block is not given' do
+    arr = [1, 2, 3, 4]
```

Comment:
- [ ] You are defining this array a lot of times. For variables like this, you can use the _:let_  to define them one time at the top of the tests. This will make your code cleaner. Do that for all variables you are using multiple times.